### DISCUSSION TITLE
[Feature] Management mode for admin

### DISCUSSION THREAD

**Discussion by Nonobis** [immich-app/immich#1789] [Feature Request]
### The feature

Hello,

Add a menu on mobile apps for administration of the server from mobile apps.

I think Manage Jobs/Server settings at minima coule be great

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by asareynolds** [comment]
Seconding this! It would be great to be able to make admin changes on mobile without needing to use a web browser.
